test/test-mongocrypt-crypto.c

```diff
@@ -282,7 +282,7 @@ _test_roundtrip_set (_mongocrypt_tester_t *tester, const char *path)
    if (!test_bin) {
       TEST_ERROR ("Failed loading test data file '%s'\n", path);
    }
-   if (test_bin->len) {
+   if (test_bin->len == 5) {
```

Comment:
I don't think we'll need this check anymore once the assertion in `_load_json_as_bson()` is fixed.

Reply:
yup